config/cms.ranking.toml.sample

```diff
@@ -0,0 +1,11 @@
+# Listening address for RankingWebServer.
+bind_address = ""
+
+# Listening port for RankingWebServer.
+http_port = 8890
+
+# Login information for adding and editing data.
+username = "usern4me"
+password = "passw0rd"
+
+# This is the end of this file.
```

Comment:
nit: I think this line is not really useful, we could remove it (same comment for other TOML files)